src/main/pages/che-7/administration-guide/proc_deploying-kubernetes-image-puller-using-openshift-templates.adoc

```diff
@@ -36,40 +36,20 @@ See xref:kubernetes-image-puller-configuration[] for more information about conf
 
 *Installing*
 
-. Clone the `kubernetes-image-puller` repository:
+. Clone and go to the `kubernetes-image-puller` repository:
```

Comment:
The not fixed version was better IMO

Reply:
Maybe "clone the `kubernetes-image-puller` repository and `cd` into it"?

Reply:
I made this change after asking what would be the preferred way to express this. Our documentation is quite inconsistent at the moment. I'll create a ticket for it later on.